### DISCUSSION TITLE
Clarification licensing regarding usage

### DISCUSSION THREAD

**Discussion by pisua** [dashingsoft/pyarmor#1974] [General]
Hello,

We want to use PyArmor for our product in order to protect our code through obfuscation. We are using an Azure CI solution and building it on demand, and we would like some clarification about your licensing plan, as we don't clearly understand which solution would fit our usage.

We want to create a Docker image that utilizes PyArmor for each build and use it in the target environment within a Docker container that can be renewed on machine restart.

We've seen multiple licensing plans: Basic, Pro, Group, and CI, but it's not very clear what they offer to meet our requirements, especially the differences between Group and CI.

Could you provide us with more information ? This allow us also to see the price for usage of PyArmor.  If you need any further details, please don't hesitate to ask.

Thanks in advance for your answer. 
Regards

**Comment by jondy** [comment]
Do you check these pages?
https://pyarmor.readthedocs.io/en/latest/licenses.html
https://pyarmor.readthedocs.io/en/latest/how-to/register.html
https://pyarmor.readthedocs.io/en/latest/how-to/ci.html
